### PR TITLE
Add top banner to trial projects with link to schedule tour

### PR DESCRIPTION
**Pull request by biyeun**
For subscriptions where `is_trial = True` this adds a banner to the top of the page with a link to the current subscription wrapping the text "x days remaining" and a button to schedule a tour that walks the user through a Hubspot form and on submission shows a Schedule Once calendar widget.

please merge this PR first: https://github.com/dimagi/commcare-hq/pull/22485

screen shot 2018-11-21 at 1 50 34 pm

review commit-by-commit. tiny commits 😸 

NOTE: inform Mack on merge of PR to monitor analytics

***this affects only SaaS production platforms***

### CODE REVIEW

File: corehq/apps/hqwebapp/templates/hqwebapp/base.html [hunk 1]
```diff
@@ -1,7 +1,11 @@
 {% load menu_tags %}{% load i18n %}{% load hq_shared_tags %}{% load cache %}{% load compress %}{% load statici18n %}<!DOCTYPE html>
 {% get_current_language as LANGUAGE_CODE %}
 {% requirejs_main %}
-<html lang="{{ LANGUAGE_CODE }}">
+<!DOCTYPE html>
```
Comment (biyeun) [inline corehq/apps/hqwebapp/templates/hqwebapp/base.html]: whoops. didn't see this on the top line. removing in a subsequent commit

File: corehq/apps/hqwebapp/templates/hqwebapp/partials/get_demo_banner.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{% load i18n %}
+{% load hq_shared_tags %}
+
+<div class="alert alert-warning alert-trial">
+  <i class="fa fa-info-circle"></i>
+  {% url 'domain_subscription_view' domain as url_sub %}
+  {% with num_trial_days_remaining|pluralize as days_plural %}
```
Comment (orangejenny) [inline corehq/apps/hqwebapp/templates/hqwebapp/partials/get_demo_banner.html]: I don't think this `days_plural` logic will necessarily pluralize correctly for translations (`día` => `dias`).

File: corehq/apps/hqwebapp/templates/hqwebapp/base.html [hunk 3]
```diff
@@ -1,7 +1,11 @@
...
+<!--[if lt IE 7]><html lang="{{ LANGUAGE_CODE }}" class="lt-ie9 lt-ie8 lt-ie7"><![endif]-->
+<!--[if IE 7]><html lang="{{ LANGUAGE_CODE }}" class="lt-ie9 lt-ie8"><![endif]-->
+<!--[if IE 8]><html lang="{{ LANGUAGE_CODE }}" class="lt-ie9"><![endif]-->
+<!--[if gt IE 8]><!--><html lang="{{ LANGUAGE_CODE }}"><!--<![endif]-->
```
Comment (orangejenny) [inline corehq/apps/hqwebapp/templates/hqwebapp/base.html]: Curious, do non-IE browsers show up as "greater than IE8"?

**Review by orangejenny — APPROVED** [review]
Feedback is minor & non-blocking.

File: corehq/apps/hqwebapp/templates/hqwebapp/base.html (hunk 3 above)
Comment (biyeun) [reply corehq/apps/hqwebapp/templates/hqwebapp/base.html]: two options. I believe this is shorthand `<!--[if gt IE 8]><!-->` so greater than IE8 or nothing

**Pull request merged**